libnetwork/config/config.go

```diff
@@ -39,7 +39,7 @@ type Config struct {
 	ClusterProvider        cluster.Provider
 	NetworkControlPlaneMTU int
 	DefaultAddressPool     []*ipamutils.NetworkToSplit
-	Scope                  datastore.Config
+	DatastoreConfig        datastore.Config
```

Comment:
And (with my other comments); this could become a string, and the location of the `lokal-kv.db`